pages/c/_cluster/auth/config/auth-config-utils.js

```diff
@@ -0,0 +1,19 @@
+import { _EDIT } from '@/config/query-params';
+import { MANAGEMENT } from '@/config/types';
+
+export const authProviders = async(store) => {
+  const rows = await store.dispatch(`management/findAll`, { type: MANAGEMENT.AUTH_CONFIG });
+  const nonLocal = rows.filter(x => x.name !== 'local');
+  const enabled = nonLocal.filter(x => x.enabled === true );
+
+  const enabledLocation = enabled.length === 1 ? {
+    name:   'c-cluster-auth-config-id',
+    params: { id: enabled[0].id },
+    query:  { mode: _EDIT }
+  } : null;
+
+  return {
+    nonLocal,
+    enabledLocation
+  };
+};
```

Comment:
This solution looks good to me and works as expected, but I think this function can just be added to the preexisting `utils/auth.js` file.

Reply:
Done! Good spot